e2e/interchaintestv8/chainconfig/chain_config.go

```diff
@@ -19,7 +19,7 @@ func IbcGoChainSpec(name, chainId string) *interchaintest.ChainSpec {
 			Images: []ibc.DockerImage{
 				{
 					Repository: "ghcr.io/cosmos/ibc-go-wasm-simd", // FOR LOCAL IMAGE USE: Docker Image Name
-					Version:    "main",                            // FOR LOCAL IMAGE USE: Docker Image Tag
+					Version:    "release-v10.0.x",                 // FOR LOCAL IMAGE USE: Docker Image Tag
```

Comment:
I wanna continue having tests against main though... Is it possible to separate the test suite into `interchaintestv10/` and rename the main directory to `interchaintest/`

Reply:
We can maybe move back to main as soon as we branch out to a release branch?
Right now, this should be our target.